marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -142,6 +142,11 @@ class Jobs {
         batchStatus: "started",
         timeStarted:  fn.currentDateTime(),
         timeEnded: "N/A",
+        hostName: xdmp.hostName(),
+        reqTimeStamp: xdmp:request-timestamp(),
```

Comment:
This needs to be the JavaScript signature. "xdmp.requestTimestamp()".

Reply:
updated syntax for dmp.requestTimestamp()